flair/models/word_tagger_model.py

```diff
@@ -5,48 +5,82 @@
 import torch
 
 import flair.nn
-from flair.data import Dictionary, Sentence, Token
+from flair.data import Dictionary, Sentence, Span, Token
 from flair.embeddings import TokenEmbeddings
 
 log = logging.getLogger("flair")
 
 
-class WordTagger(flair.nn.DefaultClassifier[Sentence, Token]):
+class TokenClassifier(flair.nn.DefaultClassifier[Sentence, Token]):
     """This is a simple class of models that tags individual words in text."""
```

Comment:
I think we shouldn't just remove the WordTagger as this breaks models using the WordTagger (Especially if they are part of a MultiTaskModel).

I would suggest, that we use a `DeprecationHelper` like specified [here](https://stackoverflow.com/a/9008509), but specify the version when we delete (let's say 0.14.0; incrementing by 2.)

That way we could give users a chance to upgrade between multiple versions.

We can also discuss this further in private

Reply:
Thanks, good idea!